junit-platform-console/src/main/java/org/junit/platform/console/tasks/DiffPrinter.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015-2024 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.console.tasks;
+
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.List;
+
+import com.github.difflib.text.DiffRow;
+import com.github.difflib.text.DiffRowGenerator;
+
+/**
+ * class provide access to printDiff function
+ */
+class DiffPrinter {
+	//print the difference of two print to out
+	static void printDiff(PrintWriter out, String expected, String actual) {
+		DiffRowGenerator generator = DiffRowGenerator.create().showInlineDiffs(true).inlineDiffByWord(true).oldTag(
+			f -> "~").newTag(f -> "**").build();
+		List<DiffRow> rows = generator.generateDiffRows(Arrays.asList(expected), Arrays.asList(actual));
```

Comment:
Don't we have to split the `expected` and `actual` Strings into lines here?

Reply:
What the difference is if we split them into lines?

Reply:
I wasn't sure. I haven't used this library before and I was just curious why it expects lists of Strings. Do you know the reason?

Reply:
I don't know the details but all their methods use `List<String>` as input.